joyq

```diff
@@ -292,8 +292,16 @@ def pipeline():
                         help="Pretty-print JSON output.")
     parser.add_argument("--sum", dest="sumvars",
                         help="Compute sum over selected element(s).")
-    parser.add_argument("--seclevel", dest='seclevel', action="store_true",
+    parser.add_argument("--tls_sec", dest='tls_sec', action="store_true",
                         help="Report security level of TLS sessions.")
+    parser.add_argument("--policy_file", dest='policy_file',
```

Comment:
Could you change these options names to something more unique such as --sec_policy_file so that they are distinguished from other options that will be added in the future?